.travis.yml

```diff
@@ -6,7 +6,7 @@ language: go
 go:
 - "1.11.x"
 env:
-  - GO111MODULE=off
+  - GO111MODULE=on
```

Comment:
Can you see if we're able to safely set `GOFLAGS=-mod=vendor` as well?

Reply:
I believe it's unnecessary, as the linting tool does it automatically?

Reply:
Oh not for the linting tool, just because Go will realise we're using vendored modules in that case.